client/disputes/evidence/index.js

```diff
@@ -621,16 +152,17 @@ export default ( { query } ) => {
 
 	return (
 		<DisputeEvidencePage
-			isLoading={ loading }
+			isLoading={ isLoading }
+			isSavingEvidence={ isSavingEvidence }
```

Comment:
isSavingEvidence is used here to set the loading state of the save evidence button 🙂

Reply:
It looks like clicking the submit button will also set loading state of the save evidence button – should an `isSubmittingEvidence` be handled separately?

Reply:
Nice catch!

Yeah, we should probably set distinct state for that 🤔